### ISSUE TITLE
{Linked:Bug2531735}[Adaptive Card > Designer > Select Host App]: Screen reader is not announcing state 'selected' for selected host in list item after activating 'Select Host' combo box.

### ISSUE THREAD

**Issue by MohitTyagi13** [microsoft/AdaptiveCards#7781]
### Target Platforms

Other

### SDK Version

1.3

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (OS Build 25188.1000)
Browser: Microsoft Edge Dev Version 106.0.1356.0 (Official build) dev (64-bit)
URLs: https://adaptivecards.io/designer/
Screen Reader: Narrator
Tools: Accessibility insight for web 

**Repro Steps:**
Open above URL in Edge Dev Browser.
Turn on Narrator 'Ctrl + Win + Enter' key.

Navigate to 'Select Host' combo box and activate it.
Observe the announcement for selected host in list item.

**Actual Result:**

Screen reader is not announcing state 'selected' for selected host in list item after activating 'Select Host' combo box.
It should be appeared as selected in list item separately.

Screen reader is announcing as 'Bot Framework Web Chat menu item 1 of 10 current' and **current** is announcing with each list items when user is navigating the list items of drop down.

**Expected Results:**
Screen reader should announce state 'selected' for selected host in list item after activating 'Select Host' combo box.

Screen reader should announce as 'Bot Framework Web Chat 1 of 10 selected'

**Observation:**

Same issue is repro'ing with NVDA in Chrome Browser.

**MAS Reference Link:**

https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02

“Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=05%7c01%7cv-raians%40microsoft.com%7c7c402eb8318b40f98eb408da5f4d1134%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637927082253567151%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=E8Iq68CSJWIrxN7fhM9bSFiR3jqBozeC%2fvo0VVHX9HI%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95915837/187194494-4327e24d-48dd-4210-ba66-c0663bdd07ec.mp4

https://user-images.githubusercontent.com/95915837/187196085-6e43d8ef-74c7-49fd-9763-01536268a7dd.mp4

### Card JSON

```JSON
N/A
```

**Comment by anna-dingler** [comment]
@MohitTyagi13 Would it be acceptable if the screen reader announced "Bot Framework WebChat RadioMenuItem Checked Current 1 of 10" and for non-selected items it announced "Outlook Actionable Messages RadioMenuItem Not Checked Current 2 of 10" etc.?

**Comment by MohitTyagi13** [comment]
@anna-dingler, it's fine but text **current** should not announced as it will be confused screen reader user.

**Comment by vagpt** [comment]
This issue is working fine on the below environment. hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1
Screen Reader: Narrator
